client/wildcard/src/components/Alert/story/Alert.story.tsx

```diff
@@ -0,0 +1,49 @@
+import { Meta } from '@storybook/react'
+import React from 'react'
+
+import { BrandedStory } from '@sourcegraph/branded/src/components/BrandedStory'
+import webStyles from '@sourcegraph/web/src/SourcegraphWebApp.scss'
+
+import { Alert } from '../Alert'
+import { ALERT_VARIANTS } from '../constants'
+
+import { AlertVariants } from './AlertVariants'
+
+const Story: Meta = {
+    title: 'wildcard/Alert',
+    decorators: [
+        story => (
+            <BrandedStory styles={webStyles}>{() => <div className="container mt-3">{story()}</div>}</BrandedStory>
+        ),
+    ],
+    parameters: {
+        component: Alert,
+        design: [
+            {
+                type: 'figma',
+                name: 'Figma Light',
+                url:
+                    'https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1563%3A196',
+            },
+            {
+                type: 'figma',
+                name: 'Figma Dark',
+                url:
+                    'https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1563%3A525',
+            },
+        ],
+    },
+}
+
+export default Story
+
+export const AllAlerts = () => (
```

Comment:
Let's use the `Story` type in places like this one.

```suggestion
export const Alerts: Story = () => (
```